### DISCUSSION TITLE
Allow to register for notifications based on post types and taxonomies

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Planner#1293] [Feature Requests]
We can register for receiving notifications clicking on the "Notify me" link for individual posts. But sometimes the user wants to automatically subscribe for all posts of a specific post type. We can give an option for that in the profile page, or elsewhere, allowing to filter by post type and taxonomies.

**Comment by stevejburge** [comment]
One more:

>  I have my posts classified by categories. I would like to enable my visitors to subscribe to new post notification by choosing all or specific categories. Then I would like PublishPress to email subscribers if a new post fits the categories that they are interested in. An added bonus would be to specify that only one email goes out every day or week. Can PublishPress do this?
